source/faq.txt

```diff
@@ -12,6 +12,27 @@ Frequently Asked Questions
    :depth: 2
    :class: singlecol
 
+How to fix MongoServerSelectionError: connect ECONNREFUSED ::1:27017 error?
+---------------------------------------------------------------------------
+
+Beginning with version 17, Node defaults to using ``IPV6``. You have a few
+options to resolve this:
```

Comment:
Identify "this" since the prior sentence refers to a default setting, not the issue.
 
```suggestion
options to resolve this error:
```

Reply:
Issue:
Seems like there are a couple solutions if I understand the second one correctly: one is to use ipv6 and the other is to use ipv4. I think it could be helpful to explain those as the two alternative paths they can take to determine whether the listed solution applies to them.

Suggestion:
Bullet 1) "If your server is running on IPv4, specify the { family: 4 } option when instantiating your MongoClient"
Bullet 2) "If you can broadcast your server on IPv6, restart your server with the ..."